Wikipedia/Code/WMFArticlePreviewTableViewCell.m

```diff
@@ -30,11 +34,22 @@ @implementation WMFArticlePreviewTableViewCell
 
 #pragma mark - Setup
 
+- (void)blurAndShowLoadingIndicator {
+    self.blurView.hidden = NO;
+    [self.activityIndicator startAnimating];
+}
+
+- (void)unblurAndHideLoadingIndicator {
+    self.blurView.hidden = YES;
+    [self.activityIndicator stopAnimating];
```

Comment:
setting `hidesWhenStopped` causes it to hide when stopped. Not sure if the logic works the other way around - it may not redrawing because its subview is hidden - but it may be running some other logic if Apple isn't checking the hidden property.